client/signup/jetpack-connect/site-url-input.jsx

```diff
@@ -34,6 +34,12 @@ export default React.createClass( {
 		return( this.translate( 'Connecting…' ) )
 	},
 
+	maybeClickIfEnter( event ) {
```

Comment:
We could be more generic and name this `handleKeyUp` in case, though it is doubtful, we need to respond to other key codes.